jsx/Tabs.js

```diff
@@ -141,6 +141,123 @@ Tabs.defaultProps = {
   updateURL: false
 };
 
+/**
+ * Allows to dynamically render vertical tabs corresponding to tab panes.
+ */
+
+class VerticalTabs extends React.Component {
```

Comment:
I don't see why this (or any javascript at all) is part of a PR about normalizing database tables..

Reply:
@driusan this addition should not have been added here because it has its own separate PR. However some front end changes will occur alongside the normalization as part of the changes on consent. at the moment they will all be added here as per my instructions for simplifying collaboration and at a later time hopefully broken down into pieces or cleaned up and renamed here to "Improvements on consents in loris"

Reply:
That doesn't simplify collaboration in any way, it makes it far more complicated. One pull request needs to do one thing.

Reply:
@driusan couldnt agree more but when 5 things depend on each other and they are in 5 different PRs it becomes complicated to quickly switch branches test the changes. 

The "Needs Work" tag will be removed when the PR is in an appropriate state for peer review

Reply:
@driusan the vertical tabs stuff was a mistake! i didnt even know that i had committed and pushed that :( the index.js also shouldnt be there. please reopen - changes to Data and formData in ConsentStatus.js were made to align with changes made to the ajax scripts as a result of normalizing the tables and renaming them to follow the recent SQL modelling conventions